### DISCUSSION TITLE
Cairo Seekbar

### DISCUSSION THREAD

**Discussion by sultanmonsur** [anddea/revanced-patches#734] [General]
Cairo Seekbar Not Working With New Patch.
YouTube Version 19.26.42
![Screenshot_2024-07-15-18-27-01-43_0635f5993b1f7a268b0ea09a0649cdab](https://github.com/user-attachments/assets/03f44195-cd42-4382-ae6b-8e0fc9a0d8ff)

**Comment by anddea** [comment]
There are some A/B tests, no bug reports will be accepted for YT above 19.16. Cairo seelbar is working for me. There's something on your side, either patch issues, or A/B.
